External/Plugins/CodeRefactor/Commands/OrganizeImports.cs

```diff
@@ -226,7 +225,24 @@ class ImportsComparerType : IComparer<MemberModel>
     {
         public Int32 Compare(MemberModel item1, MemberModel item2)
```

Comment:
The comparer could be in ASCompletion and be reused by CodeRefactor without any problem.